src/bin/flamegraph.rs

```diff
@@ -9,20 +9,61 @@ use structopt::StructOpt;
 #[derive(Debug, StructOpt)]
 #[structopt(name = "inferno-flamegraph", author = "")]
 struct Opt {
-    /// Collapsed perf output files. With no INFILE, or INFILE is -, read STDIN.
-    #[structopt(name = "INFILE", parse(from_os_str))]
-    infiles: Vec<PathBuf>,
+    // ************* //
```

Comment:
Doesn't this also need the same `_` trick as in `perf`?

Reply:
I'm thinking we eliminate my little `_` trick in `perf`. What do you think?

With `_` trick:

```
FLAGS:
        --addrs      Include raw addresses where symbols can't be found
        --all        All annotations (--kernel --jit)
        --jit        Annotate jit functions with a _[j]
        --kernel     Annotate kernel functions with a _[k]
        --pid        Include PID with process names
        --tid        Include TID and PID with process names
    -h, --help       Prints help information
    -q, --quiet      Silence all log output
    -V, --version    Prints version information
    -v, --verbose    Verbose logging mode (-v, -vv, -vvv)
```

Without `_` trick (pure alphabetical)...

```
FLAGS:
        --addrs      Include raw addresses where symbols can't be found
        --all        All annotations (--kernel --jit)
    -h, --help       Prints help information
        --jit        Annotate jit functions with a _[j]
        --kernel     Annotate kernel functions with a _[k]
        --pid        Include PID with process names
    -q, --quiet      Silence all log output
        --tid        Include TID and PID with process names
    -V, --version    Prints version information
    -v, --verbose    Verbose logging mode (-v, -vv, -vvv)
```

Reply:
I'm generally torn and the easiest "rule" to apply across all the different bins for consistency is "just alphabetical".

Reply:
I'd be okay with eliminating it here, and then either reintroduce it or find some other workaround (perhaps including changes to structopt) in a different PR.

Reply:
Cool - Just to be clear, though, we're still making a change from master. On master, the flags aren't in alphabetical order. For example, `--addr` is listed *after* `--help` because it's field name is `include_addr` (which determines ordering) and it's long attribute is `addr` (which is what is displayed from the user).